plugins/catalog-react/src/components/EntityPeekAheadPopover/EntityPeekAheadPopover.tsx

```diff
@@ -146,61 +137,58 @@ export const EntityPeekAheadPopover = (props: EntityPeekAheadPopoverProps) => {
           }}
           onMouseLeave={handleOnMouseLeave}
         >
-          <>
-            {error && <ResponseErrorPanel error={error} />}
-            <Card>
+          <Card>
+            <CardContent>
+              {error && <ResponseErrorPanel error={error} />}
               {loading && <Progress />}
-
-              <CardContent>
-                {entity && (
-                  <>
-                    <Typography color="textSecondary">
-                      {compoundEntityRef.namespace}
-                    </Typography>
-                    <Typography variant="h5" component="div">
-                      {compoundEntityRef.name}
-                    </Typography>
-                    <Typography color="textSecondary">{entity.kind}</Typography>
+              {entity && (
+                <>
+                  <Typography color="textSecondary">
+                    {entity.metadata.namespace}
```

Comment:
we know these values directly from the entity with the right casing at this point (and namespace is filled in by the server, even though it wasn't there in the original yaml file)

Reply:
😎